install_config/aggregate_logging.adoc

```diff
@@ -102,309 +88,419 @@ deployed. To control component placement, specify node selectors per component t
+xref:..aggregate_logging.adoc#aggregated-elasticsearch[Elasticsearch]
+and the xref:..aggregate_logging.adoc#aggregated-fluentd[Fluentd] sections
+before choosing parameters :
+
 [NOTE]
 ====
-If you deployed logging previously, for example in a different project, then
-it is normal for the cluster roles to fail to be created because they already exist.
+By default the Elasticsearch service uses port 9300 for TCP communication
+between nodes in a cluster.
 ====
 
-. Enable the deployer service account to create an OAuthClient (normally
-a cluster administrator privilege) for Kibana to use later when authenticating
-against the master.
-+
-====
-----
-$ oadm policy add-cluster-role-to-user oauth-editor \
-       system:serviceaccount:logging:logging-deployer <1>
-----
-<1> Use the project you created earlier (for example, *logging*) when specifying
-this service account.
-====
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
 
-. Enable the Fluentd service account to mount and read system logs by adding
-it to the *privileged* security context, and also enable it to read pod metadata
-by giving it the *cluster-reader* role:
-+
-====
-----
-$ oadm policy add-scc-to-user privileged  \
-    system:serviceaccount:logging:aggregated-logging-fluentd <1>
-$ oadm policy add-cluster-role-to-user cluster-reader \
-    system:serviceaccount:logging:aggregated-logging-fluentd <1>
-----
-<1> Use the project you created earlier (for example, *logging*) when specifying
-this service account.
-====
+ifdef::openshift-origin[]
+|`openshift_logging_image_prefix`
+|The prefix for logging component images. For example, setting the prefix to
+*openshift/origin-* creates *openshift/origin-logging-fluentd:v1.5*.
+|`openshift_logging_image_version`
+|The version for logging component images. For example, setting the version to
+*v1.5* creates *openshift/origin-logging-fluentd:v1.5*.
+endif::openshift-origin[]
 
-. Enable the Elasticsearch service account to get cluster role bindings so that
-it can verify a user's roles and allow access to operations
-logs:
-+
-====
-----
-$ oadm policy add-cluster-role-to-user rolebinding-reader \
-     system:serviceaccount:logging:aggregated-logging-elasticsearch <1>
-----
-<1> Use the project you created earlier (for example, *logging*) when specifying
-this service account.
-====
+ifdef::openshift-enterprise[]
+|`openshift_logging_image_prefix`
+|The prefix for logging component images. For example, setting the prefix to
+*registry.access.redhat.com/openshift3/* creates *registry.access.redhat.com/openshift3/logging-fluentd:latest*.
+|`openshift_logging_image_version`
+|The version for logging component images. For example, setting the version to
+*v3.5* creates *registry.access.redhat.com/openshift3/logging-fluentd:v3.5*.
+endif::openshift-enterprise[]
 
-. Ensure that port 9300 is open. By default the Elasticsearch service uses port 9300 for TCP communication between nodes in a cluster.
+|`openshift_logging_use_ops`
```

Comment:
There is a section for `openshift_logging_install_logging`, I updated it to add the uninstall portion.